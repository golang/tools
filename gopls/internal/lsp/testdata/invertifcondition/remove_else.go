package invertifcondition

import (
	"fmt"
)

func RemoveElse() {
	if true { //@suggestedfix("if true", "refactor.rewrite", "")
		fmt.Println("A")
	} else {
		fmt.Println("B")
		return
	}

	fmt.Println("C")
}
