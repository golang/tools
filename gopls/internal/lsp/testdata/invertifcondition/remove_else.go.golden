-- suggestedfix_remove_else_8_2 --
package invertifcondition

import (
	"fmt"
)

func RemoveElse() {
	if false {
		fmt.Println("B")
		return
	}

	//@suggestedfix("if true", "refactor.rewrite", "")
	fmt.Println("A")

	fmt.Println("C")
}

